scripts/standard_configure_mac.sh

```diff
@@ -43,11 +43,13 @@ rm -rf "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}/CMakeCache.txt"
 rm -rf "${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}/CMakeFiles"
 
 # Configure
-/Applications/CMake.app/Contents/bin/cmake -H"${KRATOS_SOURCE}" -B"${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}"    \
- -DCMAKE_CXX_FLAGS="${CMAKE_CXX_FLAGS} -msse3 -std=c++11 -L/usr/local/opt/llvm/lib"                         \
- -DCMAKE_C_FLAGS="${CMAKE_C_FLAGS} -msse3 -L/usr/local/opt/llvm/lib"                                        \
- -DUSE_EIGEN_MKL=OFF                                                                                        \
- -DKRATOS_GENERATE_PYTHON_STUBS=ON                                                                          \
+/Applications/CMake.app/Contents/bin/cmake                                                 \
+ -H"${KRATOS_SOURCE}"                                                                      \
+ -B"${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}"                                                  \
+ -DCMAKE_CXX_FLAGS="${CMAKE_CXX_FLAGS} -march=native -std=c++17 -L/usr/local/opt/llvm/lib" \
```

Comment:
```suggestion
 -DCMAKE_CXX_FLAGS="${CMAKE_CXX_FLAGS} -march=native -L/usr/local/opt/llvm/lib" \
```

I would remove this unless it is absolutely necessary and let CMake handle this

Reply:
Done